python/paddle/trainer_config_helpers/evaluators.py

```diff
@@ -297,9 +297,8 @@ def auc_evaluator(
 def pnpair_evaluator(
         input,
         label,
-        info,
```

Comment:
do not delete `info`, it is useful in https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/gserver/evaluators/Evaluator.cpp#L874

the info should be put into `input`.